packages/ag-charts-community/.dependency-cruiser.js

```diff
@@ -209,7 +209,7 @@ module.exports = {
             comment: "Don't use top-level export bundles internally.",
             severity: 'error',
             from: { path: 'src/.*/' },
-            to: { path: 'src/[^/]*.ts' },
+            to: { path: 'src/[^/]*.ts', pathNot: 'src/version.ts' },
```

Comment:
@alantreadway does this seem ok, or is there a better way around this? The version is used in `src/chart/agChartV2.ts`.

Reply:
Ah, on balance this seems fine - thanks for flagging.